NeosModLoader/ModLoaderConfiguration.cs

```diff
@@ -1,73 +1,113 @@
+using HarmonyLib;
+using NeosModLoader.Utility;
 using System;
+using System.ComponentModel;
 using System.IO;
+using System.Linq;
 using System.Reflection;
 
 namespace NeosModLoader
 {
 	internal class ModLoaderConfiguration
 	{
+		private static readonly Lazy<ModLoaderConfiguration> _configuration = new(LoadConfig);
 		private static readonly string CONFIG_FILENAME = "NeosModLoader.config";
 
-		private static ModLoaderConfiguration? _configuration;
+		public bool AdvertiseVersion { get; private set; } = false;
+
+		public bool Debug { get; private set; } = false;
 
-		internal static ModLoaderConfiguration Get()
+		public bool ExposeLateTypes
 		{
-			if (_configuration == null)
-			{
-				// the config file can just sit next to the dll. Simple.
-				string path = Path.Combine(GetAssemblyDirectory(), CONFIG_FILENAME);
-				_configuration = new ModLoaderConfiguration();
+			get => !HideLateTypes;
+			set => HideLateTypes = !value;
+		}
+
+		public bool ExposeModTypes
+		{
+			get => !HideModTypes;
+			set => HideModTypes = !value;
+		}
+
+		public bool HideConflicts
+		{
+			get => !LogConflicts;
+			set => LogConflicts = !value;
+		}
```

Comment:
...is there an actual reason to duplicate these...?

Reply:
> Added some inverted properties to allow setting them with flag arguments.